docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -96,6 +96,31 @@ Google Analytics, Matomo (previously Piwik), and other analytics services can be
 
 Place custom files in corresponding sub-folder under `custom/options`.
 
+**NOTE:** The files should not have a file extension, e.g. `Labels` rather than `Labels.txt`
+
+### gitignores
+
+To add custom .gitignore, add a file with existing [.gitignore rules](https://git-scm.com/docs/gitignore) in it to `custom/options/gitignore`
+
+### Labels
+
+To add a custom label set, add a file that follows [this](https://github.com/go-gitea/gitea/blob/master/options/label/Default) format to `custom/options/label`  
```

Comment:
Change this to: `...add a file that follows the [label format](https://github.com/go-gitea/gitea/blob/master/options/label/Default) to...`

Links should really describe what they're pointing to. Avoid linking from "this" etc.

Reply:
Thanks for the feedback. In hindsight this seems obvious, I don't write docs often. I'll update this, thanks!

Reply:
Something that has helped my writing for the web is accessibility guidelines. I have found [WebAIM's guidance on this matter](https://webaim.org/techniques/hypertext/link_text#text) very helpful.

Reply:
Done